src/lib/roboChimp.ts

```diff
@@ -31,5 +53,8 @@ export async function roboChimpUserFetch(userID: string) {
 		},
 		update: {}
 	});
-	return result;
+	// if (!result[clKey] || !result[levelKey]) {
+	return roboChimpSyncData(result);
```

Comment:
I'm not sure it's best to sync the cl everytime the roboChimp user is fetched.

I feel like it's going to add significant lag/delay fetching the users cl data and CL filtering on every call to roboChimpUserFetch

if you want to keep it in this function, maybe add a 'osb_last_sync' / 'bso_last_sync' to robo chimp, and if it's been more than 4 hours or something then do the CL sync